### PR TITLE
[ONNX] Add comment on test_view_dynamic_zero_dim

### PR DESCRIPTION
**Pull request by titaiwangms**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #105951
* __->__ #105950

From https://github.com/pytorch/pytorch/issues/105066. 

The case was meant to test 0 bbox generated by vision models, but the bboxes still have `.view()` operated. The case was disabled, and not supported. Added a comment to clear the potential confusion in the future. We will wait for model example to proceed on this.

### CODE REVIEW

File: test/onnx/test_fx_to_onnx_with_onnxruntime.py [hunk 1]
```diff
@@ -566,21 +566,21 @@ def forward(self, x):
...
             def forward(self, input):
                 input = input.view(-1, 2)
                 return input.view(1, -1)
 
         x = torch.ones(2)
+        # y = torch.empty(0)
         self.run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
             ViewModel(),
             (x,),
+            # additional_test_inputs=[((y,),)],
```
Comment (BowenBao) [inline test/onnx/test_fx_to_onnx_with_onnxruntime.py:583]: Is this actually being tested in current CI? I thought it was skipped by a change.

Comment (BowenBao) [reply test/onnx/test_fx_to_onnx_with_onnxruntime.py:583]: ah this is plain new code, so it was already removed lol.

**Review by BowenBao — APPROVED** [review]

**Review by thiagocrepaldi — APPROVED** [review]
LGTM. Add a TODO to uncomment the addition `y` input soon?

**Pull request closed without merging**